scarlet/initialization.py

```diff
@@ -160,7 +156,7 @@ def trim_morphology(sky_coord, frame, morph, bg_cutoff, thresh):
                 )
             )
             while boxsize < size:
-                boxsize *= 2
+                boxsize += 16  # keep box sizes quite small
```

Comment:
Are we giving up on power of 2 boxes. (great, but won't it break anything?)

Reply:
For now there is no formal reason to use them. The question is mostly memory alignment, but since we have many boxes it may not matter as much as when they are all them same.

Reply:
I'm good with this. I've seen a lot of cases where 64 pixels or 128 pixels are much too big for sources that needed slightly more than a power of 2, even those most sources are contained in 32x32 pixel boxes.